src/components/GroupSuggestions/SuggestionBox/styles.scss

```diff
@@ -2,11 +2,12 @@
 
 .group-suggestion-box {
   padding: 10px;
+  height: 250px;
 }
```

Comment:
Maybe change this to min-height? With more than four group members, this overflows the container.

<img width="427" alt="Screenshot 2020-01-28 at 10 01 18" src="https://user-images.githubusercontent.com/33627243/73245730-9e5cf680-41b5-11ea-8ee7-30b4874d1aac.png">

Reply:
Ahh true!